README.md

```
@@ -1 +1,176 @@
+
+## Getting Started
+
+### Prerequisites
+
+- Docker and Docker Desktop installed
+- Make (comes pre-installed on macOS/Linux)
+
+### Quick Start
+
+1. Clone the repository:
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)